examples/pytorch_imagenet_resnet50.py

```diff
@@ -173,6 +176,7 @@ def train(epoch):
                 output = model(data_batch)
                 train_accuracy.update(accuracy(output, target_batch))
                 loss = F.cross_entropy(output, target_batch)
+                loss.div_(allreduce_batch_size).mul_(args.batch_size)
```

Comment:
Why not just `.div_(args.batches_per_allreduce)`?  Saves one operation.

Reply:
Let's add a comment saying that this re-scaling averages gradients among sub-batches.